### PR TITLE
changes to run provided RestInit test case

### PR DESCRIPTION
**Pull request by gorodechnyj**

### CODE REVIEW

File: test/io/ably/test/rest/RestInit.java [hunk 1]
```diff
@@ -215,13 +212,20 @@ public void init_default_log_level() {
 	@Test
 	public void init_default_log_output_stream() {
 		ByteArrayOutputStream outContent = new ByteArrayOutputStream();
-		PrintStream defaultOs = System.out;
+		PrintStream newTarget = new PrintStream(outContent);
+		PrintStream oldTarget = System.out;
+		// Log level was changed in above tests, turning in back to defaults
```
Comment (paddybyers) [inline test/io/ably/test/rest/RestInit.java]: Please use the same comment style as the existing code.

File: test/io/ably/test/rest/RestInit.java [hunk 2]
```diff
@@ -11,11 +7,12 @@
 import io.ably.types.ClientOptions;
 import io.ably.util.Log;
 import io.ably.util.Log.LogHandler;
+import org.junit.Test;
 
 import java.io.ByteArrayOutputStream;
 import java.io.PrintStream;
 
-import org.junit.Test;
+import static org.junit.Assert.*;
```
Comment (paddybyers) [inline test/io/ably/test/rest/RestInit.java]: Please use the same style for imports as the existing code (ie without wildcards).

**Comment by paddybyers** [comment]
Would it be possible please to rebase these changes to the `gradle-build-rebase` branch? We're in the final stages of getting that ready to merge and it would be nice to have these tests passing.

Thanks

**Comment by gorodechnyj** [comment]
Yes, it's possible. Will look to it.
Can you add me to your chat?
28 дек. 2015 г. 14:51 пользователь "Paddy Byers" <notifications@github.com>
написал:

**Comment by gorodechnyj** [comment]
I'm closing this pull request, because I have created another one, for branch `gradle-build-rebase`
https://github.com/ably/ably-java/pull/58

**Pull request closed without merging**